### PR TITLE
Update product_strategy.md

### PR DESCRIPTION
**Pull request by calerubensteingsa**
#  #

### CODE REVIEW

**Review by michelle-rago — COMMENTED** [review]
LGTM!

**Review by michelle-rago — APPROVED** [review]

File: docs/product/product_strategy.md [hunk 1]
```diff
@@ -1,34 +1,24 @@
...
 
-## Primary, Secondary, Tertiary Users
-### Primary:
-* US-based government organizations and publicly-controlled entities who use or should use the registrar
-  * _NOTE: Segmenting our audience is a separate conversation and so this intentionally broad as placeholder_
-* CISA .gov administrators
+CISA intends to create a modern, user-centered, responsive web application to enable .gov registrants to manage their domain’s registration lifecycle, DNS settings, and useful supporting services. The registrar should be the central .gov hub for CISA, supporting registrant management, and tracking technical performance indicators for the TLD. For CISA and registrants, the registrar should help generate insights into the security of an organization’s internet-accessible systems.
 
+## Problem statements
+U.S.-based government organizations and publicly controlled entities lack a clear, usable, and efficient way to apply for, register, and manage .gov domains and related infrastructure to build public trust of their website and communications.
```
Comment (h-m-f-t) [inline docs/product/product_strategy.md:13]: I think this should change: "to build public trust in their online services and communications"

File: docs/product/product_strategy.md [hunk 2]
```diff
@@ -1,34 +1,24 @@
...
 
-## Problem Statements
-U.S.-based government organizations and publicly controlled entities lack a clear, usable, and efficient way to apply, register, and a .gov domain and related infrastructure in order to build public trust of their website and communications.
+CISA lacks a scalable, efficient, and secure method of managing the .gov TLD program that helps government agencies build public trust of their website and communications.
```
Comment (h-m-f-t) [inline docs/product/product_strategy.md:15]: Same.

**Review by h-m-f-t — CHANGES_REQUESTED** [review]
Thanks for preparing, Cale! Sorry to have missed these in the doc.

**Review by calerubensteingsa — COMMENTED** [review]
Updated (I think!)

**Review by h-m-f-t — APPROVED** [review]
Thanks!

**Pull request merged**